cdc/entry/mounter.go

```diff
@@ -219,29 +219,38 @@ func (m *mounterImpl) unmarshalRowKVEntry(tableInfo *model.TableInfo, rawKey []b
 	}, nil
 }
 
-const (
-	ddlJobListKey         = "DDLJobList"
-	ddlAddIndexJobListKey = "DDLJobAddIdxList"
-)
-
-// UnmarshalDDL unmarshals the ddl job from RawKVEntry
-func UnmarshalDDL(raw *model.RawKVEntry) (*timodel.Job, error) {
-	if raw.OpType != model.OpTypePut || !bytes.HasPrefix(raw.Key, metaPrefix) {
+// ParseJob parses the job from the raw KV entry.
+func ParseJob(tblInfo *model.TableInfo, rawKV *model.RawKVEntry, id int64) (*timodel.Job, error) {
```

Comment:
Can you add comments on the meaning of `id`? It is used to locate the column `job_meta`, but it is hard to know the purpose from this function alone.